pgx-utils/src/sql_entity_graph/postgres_ord/mod.rs

```diff
@@ -67,17 +67,21 @@ pub struct PostgresOrd {
 }
 
 impl PostgresOrd {
-    pub fn new(name: Ident, to_sql_config: ToSqlConfig) -> Self {
-        Self {
+    pub fn new(name: Ident, to_sql_config: ToSqlConfig) -> Result<Self, syn::Error> {
+        if !to_sql_config.overrides_default() {
+            crate::ident_is_acceptable_to_postgres(&name)?;
+        }
+
+        Ok(Self {
             name,
             to_sql_config,
-        }
+        })
     }
 
     pub fn from_derive_input(derive_input: DeriveInput) -> Result<Self, syn::Error> {
         let to_sql_config =
             ToSqlConfig::from_attributes(derive_input.attrs.as_slice())?.unwrap_or_default();
-        Ok(Self::new(derive_input.ident, to_sql_config))
+        Self::new(derive_input.ident, to_sql_config)
```

Comment:
Wait, how does this work? Shouldn't it still have to Ok-wrap? It's still returning a Result...?

Reply:
Oh, it's because `::new` is also a Result, okay. I missed that somehow.

Reply:
Ya! Sorry!